client/src/context/auth/AuthState.js

```diff
@@ -20,13 +20,23 @@ export const useAuth = () => {
     return [state, dispatch];
 };
 
+var route1 = '';
+var route2 = '';
+if(process.env.NODE_ENV === 'production') {
+    route1 = '/api/auth';
```

Comment:
maybe route1 and route2 can be renamed with something more related like route_auth and route_employer.

Reply:
Yes I will make that change. This was temporary to make sure this worked. I may end up making a .env file that holds these routes